src/plugin.json

```diff
@@ -29,7 +29,7 @@
   },
 
   "dependencies": {
-    "grafanaVersion": "7.0.0",
+    "grafanaVersion": "8.0.0",
```

Comment:
surprisingly -- this setting does not do anythign!  To actually change the version you need "grafanaDependency":
https://github.com/grafana/iot-sitewise-datasource/blob/main/src/plugin.json#L41

but I think we should try to find a path that works for both 7 and 8